src/OrchardCore.Modules/OrchardCore.ReCaptcha/Views/Items/ReCaptchaPart.Fields.Edit.cshtml

```diff
@@ -2,11 +2,9 @@
 @model OrchardCore.ReCaptcha.Forms.ReCaptchaPartViewModel
 @{
     var settingsUrl = Url.Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = OrchardCore.ReCaptcha.Drivers.ReCaptchaSettingsDisplayDriver.GroupId });
-    var successMessageTemplate = "<a {0}>ReCaptcha settings</a> are configured.";
-    var warningMessageTemplate = "<a {0}>ReCaptcha settings</a> are not configured. The captcha will not be displayed.";
     var hyperLinkAttributes = $"class=alert-link target=_blank href={settingsUrl}";
```

Comment:
I bet this string doesn't have the `"` because you realized that the translation would then encode them. Which show that there is an issue actually. The solution is to wrap this string in an HtmlString instead, such that the localization won't encode it. Or simplify the logic to inject a standard link to the recaptcha settings in the `if`, only if they are not configured.

Reply:
Will have a look.